src/components/HTMLEngineProvider/HTMLRenderers/PreRenderer/index.native.js

```diff
@@ -1,25 +1,25 @@
 import React from 'react';
-import {ScrollView} from 'react-native-gesture-handler';
-import {View} from 'react-native';
-import _ from 'underscore';
 import withLocalize from '../../../withLocalize';
 import htmlRendererPropTypes from '../htmlRendererPropTypes';
+import BasePreRenderer from './BasePreRenderer';
 
-const PreRenderer = (props) => {
-    const TDefaultRenderer = props.TDefaultRenderer;
-    const defaultRendererProps = _.omit(props, ['TDefaultRenderer']);
+class PreRenderer extends React.Component {
+    constructor(props) {
+        super(props);
 
-    return (
-        <ScrollView horizontal>
-            <View onStartShouldSetResponder={() => true}>
-                <TDefaultRenderer
-                     // eslint-disable-next-line react/jsx-props-no-spreading
-                    {...defaultRendererProps}
-                />
-            </View>
-        </ScrollView>
-    );
-};
+        this.ref = React.createRef(null);
```

Comment:
Are we using this ref anywhere? This can be a functional component.

Reply:
@parasharrajat okay i eliminated the ref here and convert class to functional component here.
Comitted 👍 . PR is ready.